pkgs/desktops/plasma-5/3rdparty/kwin/kde-rounded-corners.nix

```
@@ -0,0 +1,49 @@
+{ pkgs, qtbase, qttools, qtx11extras, kconfig, kcoreaddons, ki18n,
+kio, kglobalaccel, kinit, kwin, xorg, libepoxy, lib }:
```

Please order the arguments better and place on separate lines

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, cmake
, extra-cmake-modules

, qtbase
, qttools
, qtx11extras
, kconfig
, kcoreaddons
, ki18n
, kio
, kglobalaccel
, kinit
, kwin
, xorg
, libepoxy
}:
```